lightblue/src/main/java/org/esbtools/eventhandler/lightblue/LightblueNotificationRepository.java

```diff
@@ -80,10 +78,14 @@ public LightblueNotificationRepository(LightblueClient lightblue, String[] entit
             DataBulkRequest updateEntities = new DataBulkRequest();
```

Comment:
A single update request with a query containing the _ids of docs to be updated would perform better here, instead of bulk request.